interchange/DeviceResources.capnp

```diff
@@ -89,6 +107,8 @@ struct Device {
   constraints    @13 : Constraints;
   lutDefinitions @14 : LutDefinitions;
   parameterDefs  @15 : ParameterDefinitions;
+  pipTimings     @16 : List(PIPTiming);
+  nodeTimings    @17 : List(NodeTiming);
```

Comment:
Maybe instead of node timings we should have wire timings? This could allow as to build timing graphs that model distance dependent delays.